target/linux/ramips/image/rt305x.mk

```diff
@@ -651,6 +651,18 @@ define Device/sl-r7205
 endef
 TARGET_DEVICES += sl-r7205
 
+define Device/tew-638apb
+  DTS := TEW-638APB
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := $(ralink_default_fw_size_4M)
+  IMAGES += factory.bin
+  IMAGE/factory.bin := $$(sysupgrade_bin) | check-size $$$$(IMAGE_SIZE) | \
```

Comment:
Would you please try the following sysupgrade.bin code. It should do the job:

```
IMAGE/sysupgrade.bin := $$(sysupgrade_bin) | umedia-header 0x026382 | append-metadata | check-size $$$$(IMAGE_SIZE)
```

If a sysupgrade.bin generated this way works with the Trendnet webinterface firmware upgrade page, drop your factory.bin.